### PR TITLE
feat: add openziti bind support to ui

### PR DESCRIPTION
**Pull request by dovholuknf**
This PR allows the UI to be OpenZiti enabled for binding services as well

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)

### CODE REVIEW

File: internal/service/httpserver.go [hunk 1]
```diff
@@ -125,8 +133,64 @@ func (b *HttpServer) BootstrapHandler(
...
 		}()
 
+		var ln net.Listener
+		listenMode := strings.ToLower(b.cfg.Service.SecurityOptions[bscfg.SecurityModeKey])
+		switch listenMode {
+		case zerotrust.ZeroTrustMode:
+			ozUrl := b.cfg.Service.SecurityOptions["OpenZitiController"]
+			var zctx ziti.Context
+			var authErr error
+			if os.Getenv("SERVICE_SECURITYOPTIONS_OPENZITIAUTHMETHOD") == "identity" {
```
Comment (dovholuknf) [inline internal/service/httpserver.go:143]: this should eventually  probably be replaced by using the standard configuration options but these two keys need to be added to the "default" and I can't remember where that is :)

File: internal/service/httpserver.go [hunk 2]
```diff
@@ -125,8 +133,64 @@ func (b *HttpServer) BootstrapHandler(
 			b.isRunning = false
 		}()
 
+		var ln net.Listener
+		listenMode := strings.ToLower(b.cfg.Service.SecurityOptions[bscfg.SecurityModeKey])
```
Comment (dovholuknf) [inline internal/service/httpserver.go:137]: all this functionality should also be moved to `go-mod-bootstrap` or some other common place. this code is largely simlar to code in [`app-functions-sdk-go/internal/webserver/server.go`](https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/internal/webserver/server.go#L133-L179)

**Review by cloudxxx8 — APPROVED** [review]
LGTM

File: internal/service/httpserver.go (hunk 2 above)
Comment (cloudxxx8) [reply internal/service/httpserver.go:137]: yes, we should do it when we have a chance

**Pull request merged**